src/main/java/org/candlepin/subscriptions/capacity/CandlepinPoolCapacityMapper.java

```diff
@@ -77,6 +77,7 @@
             capacity.setBeginDate(pool.getStartDate());
             capacity.setEndDate(pool.getEndDate());
             capacity.setServiceLevel(sla);
+            capacity.setSku(pool.getName());
```

Comment:
Actually, is there some reason this:
```suggestion
            capacity.setSku(pool.getProductId());
```
shouldn't work?